### PR TITLE
feat: show account username suffices

### LINKED ISSUE #1409: Show all username suffixes
Now that we have a variety of namespaces with active registration in production, we should revisit the decision to hide the `id.stx` in the UI so users have a more consistent experience.

Here for example you can see the `.btc` and `.stx` suffixes but not id.stx:

Screen Shot 2021-07-08 at 15 09 52

### PR DESCRIPTION
**Pull request by markmhendrickson**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/1992940186).

Resolve https://github.com/hirosystems/stacks-wallet-web/issues/1409

# Before

image

# After

image

### CODE REVIEW

**Comment by markmhendrickson** [comment]
@kyranjamie any idea why the typecheck is failing here re: parameters?

This function is used elsewhere already with just one parameter. E.g. https://github.com/hirosystems/stacks-wallet-web/blob/4925bad0400daf0d8735ad2ab9ae5df330d6e28c/src/app/features/current-account/current-account-avatar.tsx#L20

**Comment by markmhendrickson** [comment]
I just realized @kyranjamie is out. Redirecting to @fbwoolf 🙏

**Comment by fbwoolf** [comment]
> I just realized @kyranjamie is out. Redirecting to @fbwoolf 🙏

It looks like you deleted the `useMemo` dependency array. I can fix this for you?

File: src/app/common/hooks/account/use-account-names.ts [hunk 1]
```diff
@@ -16,5 +16,5 @@ export function useCurrentAccountDisplayName() {
 
 export function useAccountDisplayName(account: SoftwareWalletAccountWithAddress) {
   const names = useGetAccountNamesByAddressQuery(account.address);
-  return useMemo(() => names[0] ?? getAccountDisplayName(account), [account, names]);
+  return useMemo(() => names[0] ?? getAccountDisplayName(account));
```
Comment (fbwoolf) [inline src/app/common/hooks/account/use-account-names.ts]: Is there a reason you removed the dependency array here?

Comment (markmhendrickson) [reply src/app/common/hooks/account/use-account-names.ts]: Hm nope, not sure why I did that in hindsight. I've pushed a new commit. 🙏

**Review by fbwoolf — APPROVED** [review]

**Review by kyranjamie — APPROVED** [review]
Great to see you pushing commits @markmhx ✨

**Pull request merged**